browser/js/common/factories/ItemFactory.js

```diff
@@ -18,6 +18,24 @@ app.factory('ItemFactory', function($http) {
 			return res.data;
 		});
 	};
+
```

Comment:
in general, can we rename things to itemId instead of productId? to remain consistent with the model names.